eng/ApiListing.targets

```diff
@@ -30,6 +18,24 @@
     <MakeDir Directories="$(_RefSourceOutputPath)"/>
   </Target>
 
+  <Target Name="_SetExportApiInnerTarget">
+    <PropertyGroup>
+      <InnerTargets>ExportApi</InnerTargets>
+    </PropertyGroup>
+  </Target>
+
+  <Target 
+    Name="ExportApiCrossTarget"
+    DependsOnTargets="_SetExportApiInnerTarget;DispatchToInnerBuilds"
+    Condition="'$(IsCrossTargetingBuild)' == 'true'" />
+
+  <Target 
+    Name="ExportApiInner"
+    DependsOnTargets="Build;RemoveExistingListings;GenerateReferenceAssemblySource"
```

Comment:
Unless I'm not understanding something this approach still forces a build to happen for all the targets.

Reply:
The condition on   `<Target Name="ExportApi" DependsOnTargets="ExportApiCrossTarget;ExportApiInner" Condition="'$(GenerateAPIListing)' == 'true'" />` skips the inner build entirely for projects that don't need it.